svc-bip-api/src/main/java/gov/va/vro/bip/service/BipApiService.java

```diff
@@ -308,25 +324,16 @@ HttpHeaders getBipHeader(String externalUserId, String externalKey) throws BipEx
     bipHttpHeaders.setContentType(MediaType.APPLICATION_JSON);
 
     Claims claims = bipApiProps.toCommonJwtClaims(externalUserId, externalKey);
-    String issuer = bipApiProps.getClaimIssuer();
     String secret = bipApiProps.getClaimSecret();
 
     // Define the signing key
     byte[] signSecretBytes = secret.getBytes(StandardCharsets.UTF_8);
     Key signingKey = new SecretKeySpec(signSecretBytes, "HmacSHA256");
 
-    // Set the expiration as an example (e.g., 1 hour from now)
-    long currentTimeMillis = System.currentTimeMillis();
-    Date expiryDate = new Date(currentTimeMillis + 3600000);
-
     // Build the JWT
     String jwt =
         Jwts.builder()
             .claims(claims)
-            .issuer(issuer)
-            .subject("Claim")
-            .issuedAt(new Date(currentTimeMillis))
-            .expiration(expiryDate)
```

Comment:
What's the motivation behind removing these fields on this Jwt builder but adding them in BipApiProps::toCommonJwtClaims?

Reply:
Felt cleaner to just build all the claims in one place.

Reply:
Honestly, I'd rip that all into another utility class to build a JWT with the BipApiProps class just used to populate values, but for now this is better.

Reply:
Fair, LGTM